apps/server/src/modules/user-login-migration/controller/user-login-migration-rollback.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Authenticate, CurrentUser, ICurrentUser } from '@modules/authentication';
+import { Controller, HttpCode, HttpStatus, Param, Post } from '@nestjs/common';
+import {
+	ApiForbiddenResponse,
+	ApiNoContentResponse,
+	ApiNotFoundResponse,
+	ApiTags,
+	ApiUnauthorizedResponse,
+	ApiUnprocessableEntityResponse,
+} from '@nestjs/swagger';
+import { UserLoginMigrationRollbackUc } from '../uc';
+import { UserIdParams } from './dto';
+
+@ApiTags('UserLoginMigration Rollback')
+@Controller('user-login-migrations')
+@Authenticate('jwt')
+export class UserLoginMigrationRollbackController {
+	constructor(private readonly userLoginMigrationRollbackUc: UserLoginMigrationRollbackUc) {}
+
+	@Post('/users/:userId/rollback-migration')
```

Comment:
Why did you created a new controller for this? Do you think we have to implement more rollback meachnisms :/ ? 
Maybe for all /users/ ? If that is the case i could understand to put the rollback thing in a new controller